proposed/http-handlers/request-handlers-meta.md

```diff
@@ -275,6 +276,10 @@ and a request handler and must return a response. The middleware may:
 - Create and return a response without passing the request to the request handler,
   thereby handling the request itself.
 
+It is expected that a middleware dispatching system will use the request handler
+to delegate response creation to the next available middleware. The last middleware
+would then act as a domain gateway to execute application code.
```

Comment:
This paragraph... doesn't quite make sense.  I can't wrap my head around what it's saying.  Let me try to rephrase and tell me if I'm in the right ballpark:

```
When delegating from one middleware to the next in a sequence, a dispatching system should use an intermediary request handler as a way to link two middlewares together.  The final or innermost middleware acts as a gateway to application code that will, in most cases, be responsible for response generation.
```

Is that accurate?  (Feel free to steal liberally if so.)  Should that be in the main spec in some form?

Reply:
It's correct, except that the "application code that will, in most cases, be responsible for response generation" will likely either be the middleware itself, or the a _request handler_. I'll see if I can come up with a re-phrase that makes sense.